### PR TITLE
using dispatch hook

### PR DESCRIPTION
**Pull request by PavanDongare**
1) handleApiErrors : doesn't seem to belong to errorhandler component. Had problems using dispatch hook in it
2) ErrorBoundary : seems 2 components declared in same file. 1) errorBoundry 2) ErrorNotification .. gives error on using dispatch.

These 2 needs to resolve before useDispatch here. Rest occurrences covered.

### CODE REVIEW

File: simplq/src/components/ErrorHandler.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import PageNotFound from './pages/PageNotFound';
 import { setErrorNotifOpen } from '../store/appSlice';
 import { store } from '../store'; // TODO: Use Hooks
 
+// ToDo: make functional
 export class ErrorBoundary extends React.Component {
   constructor(props) {
```
Comment (maaverik) [inline simplq/src/components/ErrorHandler.jsx:13]: Do you know if it's possible to get the errorBoundary to work if it's made functional? Granted, it doesn't work right now, but I haven't seen a functional implementation.

**Review by maaverik — APPROVED** [review]

File: simplq/src/components/ErrorHandler.jsx (hunk 1 above)
Comment (PavanDongare) [reply simplq/src/components/ErrorHandler.jsx:13]: I guess making separate component file might help. Will try next.

**Comment by PavanDongare** [comment]
Remaining/ To do : - separating components in errorBoundry file.

File: simplq/src/components/ErrorHandler.jsx (hunk 1 above)
Comment (PavanDongare) [reply simplq/src/components/ErrorHandler.jsx:13]: @maaverik  after spending an hour trying separating components, came across a statement on react doc site:-
"Only class components can be error boundaries." 😑

Comment (maaverik) [reply simplq/src/components/ErrorHandler.jsx:13]: Ah. Well, that settles it. Now we just have to figure out why it still doesn't work.

**Pull request merged**